kotlin-html-generator/src/main/kotlin/dev/scottpierce/html/generate/model/Element.kt

```diff
@@ -38,6 +50,12 @@ sealed class Element(
                 callingContext = Context.Body,
                 childrenContext = Context.Body
             ),
+            Normal(
+                tagName = "em",
+                callingContext = Context.Body,
+                childrenContext = Context.Body,
+                supportedAttributes = listOf()
+            ),
```

Comment:
@Tmpod This is the only file that I changed. Running the generator takes care of the rest, in case you want to add any elements in the future.

Reply:
Oh ok cool. Thanks!

Reply:
What do you mean with `generator`? I've only started using the lib a couple of days ago, so I'm not familiar with its structure.